assets/promptflow/models/how-to-use-functions-with-chat-models/asset.yaml

```diff
@@ -0,0 +1,4 @@
+extra_config: model.yaml
```

Comment:
how-to-use-functions-with-chat-models the naming is not consistent with others. Is this a template? Can we rename it to template-functions?

Reply:
It is not a template. It shows in the gallery:
![image](https://github.com/Azure/azureml-assets/assets/49388944/58bf8f4a-7c78-4b17-be8b-10f599337067)
Just keep it as the example shown. And later we can discuss whether need to change the example name or not.